projects/packages/publicize/src/rest-api/class-scheduled-actions-controller.php

```diff
@@ -151,7 +173,15 @@ public function get_item_schema() {
 				),
 				'share_date'    => array(
 					'type'        => 'string',
-					'description' => __( 'ISO 8601 formatted date for the action.', 'jetpack-publicize-pkg' ),
+					'description' => __( 'ISO 8601 formatted date for the action.', 'jetpack-publicize-pkg' ) . ' ' . sprintf(
+						/* translators: %s is the new field name */
+						__( 'Deprecated in favor of %s.', 'jetpack-publicize-pkg' ),
+						'timestamp'
+					),
+				),
+				'timestamp'     => array(
+					'type'        => 'string',
```

Comment:
I think this should be integer

Reply:
Fixed in 5ba2965a56cd569f51acaaacd94827466fe5133d. Thanks